various-receipes/db-optimization/docker-compose-mysql.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.6'
+services:
+  database:
+    image: mysql:8
+    command: --default-authentication-plugin=mysql_native_password --innodb_flush_log_at_trx_commit=2 --sync-binlog=0
+    environment:
+      - MYSQL_ROOT_PASSWORD=example
+      - MYSQL_USER=myuser
+      - MYSQL_PASSWORD=myuserpassword
+      - MYSQL_DATABASE=shop
+    container_name: 'mysql-for-testing'
+    ports:
+      - '33060:3306'
+    # Do not use Memory engine, use RAM folder (tmpfs) instead (https://github.com/testjavascript/nodejs-integration-tests-best-practices/issues/9#issuecomment-710674437)
+    tmpfs: /var/lib/mysql
```

Comment:
Also map the data log to tmpfs, many IO goes to DB log (not logger rather the data log)?

This comments applies to all DB 👆

Reply:
> By default no log files are enabled in MYSQL.

https://stackoverflow.com/a/29409969/3266867

Reply:
@mikicho This suggestion is not related with observability logging rather write-ahead logging (WAL)